python/ribasim/ribasim/input_base.py

```diff
@@ -30,31 +55,144 @@ def exists(connection: Connection, name: str) -> bool:
     return result is not None
 
 
-class TableModel(BaseModel):
-    class Config:
-        validate_assignment = True
+PathOrStr = Union[Path, str]
```

Comment:
```suggestion
PathOrStr = Path | str
```

Tbh, with the new syntax, I don't think there's a need for an alias anymore